legacy/c3_trajectory_generator/README.md

```diff
@@ -0,0 +1,9 @@
+## C3_Trajectory_Generator
+
+## Waypoint Validity
+The following are the error codes that waypoint validity returns:
+- OCCUPIED = 99
+- UNKNOWN = 50
+- UNOCCUPIED = 0
+- TOO_HIGH = 1
```

Comment:
I think this should be `ABOVE_SURFACE`. I didn't understand what this was for until I looked at the  `WaypointValidity::is_waypoint_valid` implementation.